nservicebus/bridge/configuration_monitoring_bridge_[2,).partial.md

```diff
@@ -0,0 +1,39 @@
+
+Heartbeats can be configured on a per bridge transport basis.  For each bridge transport that should send hearbeats, use the `.SendHeartbeatTo` as follows:
+
+snippet: configure-heartbeats
+
+NOTE: `ServiceControl_Queue` is a placeholder for the name of the ServiceControl input queue. The name of the ServiceControl input queue matches the [ServiceControl service name](/servicecontrol/installation.md#servicecontrol-plugins) configured in the ServiceControl Management application.
+
+#### Heartbeat interval
+
+Heartbeat messages are sent at a default frequency of 10 seconds. As shown above, the frequency may be overridden for each endpoint.
+
+NOTE: The frequency must be lower than the [`HeartbeatGracePeriod`](/servicecontrol/creating-config-file.md#plugin-specific-servicecontrolheartbeatgraceperiod) in ServiceControl.
+
+#### Time-To-Live (TTL)
+
+Heartbeat messages are sent with a default TTL of four times the frequency. As shown above, the TTL may be overridden for each endpoint. See the documentation for [expired heartbeats](/monitoring/heartbeats/expired-heartbeats.md) for more information.
+
+#### Identifying scaled-out endpoints
+
+When heartbeats are being used in a [scaled-out Messaging Bridge that is using competing consumers](/nservicebus/bridge/performance.md#scaling-out-competing-consumers), each bridge transport must be configured with a unique name (see [Configuring transports](#auditing-configuring-transports)). The bridge transport name is used in the heartbeat messages so ServiceControl can track of all instances and identify which instance sent a given heartbeat message.
+
+### Custom Checks
+
+Version 2.2 of the Messaging Bridge adds support for Custom Checks.  The Custom Checks feature enables Messaging Bridge health monitoring by running custom code and reporting status (success or failure) to a ServiceControl instance. See the Custom [Checks plugin documentation](/monitoring/custom-checks/) for more information.  For documentation on how to write custom checks, see the [Writing Custom Checks document](/monitoring/custom-checks/writing-custom-checks.md)
+
+The Custom Checks feature can be enabled per bridge transport by using `ReportCustomChecksTo` as follow:
+
+snippet: configure-custom-checks
+
+NOTE: Each custom check is executed once and the result is sent to each bridge transport that is configured to report custom checks. When configured to [Bridge platform queues](#bridging-platform-queues), it is only necesary to report custom checks on one of the bridge transports, otherwise a custom check result will be reported multiple times to the same platform instance.
```

Comment:
```suggestion
NOTE: Each custom check is executed once, and the result is sent to each bridge transport configured to report custom checks. When configured to [Bridge platform queues](#bridging-platform-queues), it is only necessary to report custom checks on one of the bridge transports. Otherwise, a custom check result will be reported multiple times to the same platform instance.
```